lib/node/rpc.js

```diff
@@ -259,8 +259,128 @@ class RPC extends RPCBase {
   }
 
   async help(args, _help) {
-    if (args.length === 0)
-      return 'Select a command.';
+    if (args.length === 0) {
```

Comment:
Do you know about template strings?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

They can make creating a multiline string easier, but what you have definitely works

Reply:
I think I started with this method, but it "literally" output all this whitespace at the top or something that didn't look great.